base_custom_info/views/custom_info_value_view.xml

```diff
@@ -1,19 +1,53 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
+<!-- Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+<odoo>
 
 <record id="base_custom_info_value_tree" model="ir.ui.view">
     <field name="name">base.custom.info.value.tree</field>
     <field name="model">custom.info.value</field>
     <field name="arch" type="xml">
         <tree string="Custom Property Values">
```

Comment:
Actually I thought that letting users put additional properties could help under some circumnstances. What do you think?